addons/controllers/cpi/vspherecpiconfig_utils.go

```diff
@@ -84,27 +78,9 @@ func (r *VSphereCPIConfigReconciler) mapCPIConfigToDataValuesNonParavirtual( //
 		d.NoProxy = cluster.Annotations[pkgtypes.NoProxyConfigAnnotation]
 	}
 
-	// derive nsxt related configs from cluster variable
```

Comment:
Is it possible to keep this. Or remove it in a different PR, so that in the future we could revert it if we decide to consume parameters from Cluster variables

Reply:
I am not sure if it is safe to keep them. But they will never be Cluster variables since it is addon related configurations.

Reply:
We already designed to expose kubevipCloudProvider configuration in Cluster.spec.topology.variables. The experience would be bad if customer has to modify several CRDs to create a single Cluster

Reply:
Ok you can remove it then